### DISCUSSION TITLE
CentOS 9 Stream / EL9 ?

### DISCUSSION THREAD

**Discussion by ghost** [rabbitmq/rabbitmq-server#5132] [Other] [answered]
The official download/install page says CentOS 9 Stream is supported, however, I don't see any el9 RPMs, what's up with that?

**Comment by michaelklishin** [comment, marked as answer]
Just like there is only one RabbitMQ package for Debian that works on all supported distributions,
we have seen any reason to produce separate packages for CentOS Stream 9 because the one we already produce for CentOS Stream 8 is perfectly functional.

Our Erlang RPM has a separate CentOS Stream 9 package since Erlang is all native code. RabbitMQ does not have any.
